src/stratis_cli/_actions/_data.py

```diff
@@ -207,29 +209,68 @@
 DBUS_TIMEOUT_SECONDS = 120
 
 
+def get_timeout(value):
+    """
+    Turn an input str or int into a float timeout value.
+
+    :param value: the input str or int
+    :type value: str or int
+    :raises StratisCliEnvironmentError:
+    :returns: float
+    """
+    try:
+
+        timeout_int = int(value)
+
+    except ValueError:
+        raise StratisCliEnvironmentError(
+            "The timeout value you provided is not an integer."
+        )
+
+    # Ensure the integer is not too small
+    if timeout_int < -1:
+        raise StratisCliEnvironmentError(
+            "The timeout value you provided is smaller than the smallest acceptable value, -1."
+        )
+
+    # Ensure the integer is not too large
+    if timeout_int > (1 << 31) - 1:
+        raise StratisCliEnvironmentError(
+            "The timeout value you provided exceeds the largest acceptable value, 2147483647."
+        )
+
+    # Convert from milliseconds to seconds
+    timeout_float = timeout_int / 1000
```

Comment:
You can return a whole expression, and it's a bit more succinct:

```
return timeout_int / 1000
```

is a bit better than these two lines.